fx: 430
fy: 430
cx: 352
cy: 236
kx1: 0.00000
kx2: 0.000
ky1: 0.00000
ky2: 0.0
theta_calib: 0.00
szx: 0.4
szy: 0.4
szLED: 0.08
thsz: 1
thd: 400
nx: 704
ny: 472
#objQuad: [0.22,0.0,0.0, 0.0,0.22,0.0, 0.08,0.08,0.00,0.0,0.0,0.25, 0.04,0.04,0.04]
#objQuad: [0.20,0.0,0.075, 0.0,0.20,0.075, 0.0212,0.0212,0.075, 0.0,0.0,0.21, 0.064,0.064,0.025]
objQuad: [0.0212,0.0212,0.075, 0.0,0.20,0.075, 0.20,0.0,0.075, 0.0,0.0,0.21, 0.064,0.064,0.025]
dt_min: 0.1

